scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -157,45 +159,50 @@ protected static void start(CommandLine commandLine) throws Exception {
         hsqldbServer = new SchedulerHsqldbStarter();
         hsqldbServer.startIfNeeded();
 
-        String rmUrl = getRmUrl(commandLine);
+        rmURL = getRmUrl(commandLine);
+        //        rmURL = rmUrl;
         setCleanDatabaseProperties(commandLine);
         setCleanNodesourcesProperty(commandLine);
 
-        rmUrl = connectToOrStartResourceManager(commandLine, rmUrl);
+        rmURL = connectToOrStartResourceManager(commandLine, rmURL);
 
         if (commandLine.hasOption("rm-only")) {
             return;
         }
 
-        SchedulerAuthenticationInterface schedulerAuthenticationInterface = startScheduler(commandLine, rmUrl);
-
-        schedAuthInter = schedulerAuthenticationInterface;
-        rmURL = rmUrl;
+        schedulerURL = getSchedulerUrl(commandLine);
+        if (!commandLine.hasOption("scheduler-url")) {
```

Comment:
what does it mean to pass an option like scheduler-url ? Can you make an example?